main.tf

```diff
@@ -112,9 +112,9 @@ module "configure_project" {
   watsonx_project_description = var.watsonx_project_description
   watsonx_project_tags        = var.watsonx_project_tags
```

Comment:
Change `watsonx_project_tags` to `project_tags`

Reply:
Done the changes.